### ISSUE TITLE
server/getMemoryTrend

### ISSUE THREAD

**Issue by hanahmily** [apache/incubator-skywalking-ui#111]
## TL;DR;

No result

## Input

```graphql
{
    "variables": {
        "duration": {
            "start": "2017-08-03", 
            "end": "2017-08-05", 
            "step": "DAY"
        }, 
        "serverId": "2"
    }, 
    "query": "
  query Application($serverId: ID!, $duration: Duration!) {
    getMemoryTrend(serverId: $serverId, duration: $duration) {
      heap
      maxHeap
      noheap
      maxNoheap
    }
  }
"
}
```

## Output

```graphql
{"data":{"getMemoryTrend":{"heap":[0,0,0],"maxHeap":[0,0,0],"noheap":[0,0,0],"maxNoheap":[0,0,0]}}}
```

**Comment by peng-yongsheng** [comment]
Execute “JVMMetricServiceHandlerTestCase.java” to mock JVM metric data.

**Comment by hanahmily** [comment]
## TL;DR;

There is no "noheap" data.

## Output

```graphql

"getMemoryTrend": {
            "heap": [
                0, 
                15,
                0,
            ], 
            "maxHeap": [
                0, 
                50,
                0,
            ], 
            "noheap": [
                0, 
                0, 
                0
            ], 
            "maxNoheap": [
                0, 
                0, 
                0
            ]
        }
```

**Comment by peng-yongsheng** [comment]
#### INPUT
```
{
  getMemoryTrend(serverId: 2, duration: {start: "2017-06-01 1002", end: "2017-06-01 1005", step: MINUTE}) {
    heap, maxHeap, noheap, maxNoheap
  }
}
```

#### OUTPUT
```
{
  "data": {
    "getMemoryTrend": {
      "heap": [
        60,
        60,
        60,
        60
      ],
      "maxHeap": [
        1,
        1,
        1,
        50
      ],
      "noheap": [
        60,
        60,
        60,
        60
      ],
      "maxNoheap": [
        1,
        1,
        1,
        50
      ]
    }
  }
}
```

**Comment by hanahmily** [comment]
Transfer to https://github.com/apache/incubator-skywalking/issues/855
